backend/climateconnect_api/utility/email_setup.py

```diff
@@ -272,3 +285,50 @@ def remove_contact_from_list(contact_id, list_id):
         ]
     }
     mailjet_api.contact_managecontactslists.create(id=contact_id, data=data)
+
+
+def send_email_reminder_for_unread_notifications(
+    user: User,
+    user_notifications: List[UserNotification]
+):
+    total_notifications = user_notifications.count()
+    language_code = get_user_lang_code(user=user)
+    subject_by_language = {
+        "en": f"You have {total_notifications} unread notifications",
+        "de": f"Sie haben {total_notifications} ungelesene Benachrichtigungen"
+    }
+    subject = subject_by_language.get(language_code, "en")
+    website_link_by_language = {
+        "en": "https://climateconnect.earth/inbox",
+        "de": "https://climateconnect.earth/de/inbox"
+    }
+    website_link = website_link_by_language.get(language_code, "en")
+    email_text_by_language = {
+        "en": f"<p>Dear {user.first_name},</p><p>You have {total_notifications} new notifications. Please <a href={website_link}>click here</a> to check your inbox.</p><p>See you soon,</p><p>The Climate Connect Team</p>",  #NOQA
+        "de": f"<p>Liebe {user.first_name},</p><p>Sie haben {total_notifications} neue Benachrichtigungen. Bitte <a href={website_link}>klicken Sie hier</a>, um Ihren Posteingang zu überprüfen.</p><p>Bis bald,</p><p>Deine Climate Connect Team</p>"  #NOQA
```

Comment:
Is this advantageous over making an (unstyled) mailjet template? This seems a bit more difficult to work with to me.
Suggestion for the text:

```suggestion
        "en": f"<p>Dear {user.first_name},</p><p>You have {total_notifications} new messages. Please respond to the people who reached out because we can only limit climate change if we work together and exchange knowledge.</p> <p><a href={website_link}>Click here</a> to check your inbox.</p><p>See you soon,</p><p>The Climate Connect Team</p>",  #NOQA
        "de": f"<p>Hallo {user.first_name},</p><p>Du hast {total_notifications} ungelesene Nachrichten von anderen Klimaschützer*innen. Bitte beantworte die Nachrichten, denn gemeinsam und durch Zusammenarbeit und Wissensaustausch können wir das 1,5 Grad Ziel erreichen.</p><p><a href={website_link}>Klicke hier</a>, um deinen Posteingang anzusehen.</p><p>Bis bald,</p><p>Deine Climate Connect Team</p>"  #NOQA
```